packages/neon/neon/lib/src/pages/login.dart

```diff
@@ -56,48 +56,27 @@ class _LoginPageState extends State<LoginPage> {
                     branding.name,
                     style: Theme.of(context).textTheme.titleLarge,
                   ),
-                  const SizedBox(
-                    height: 20,
-                  ),
                   if (branding.showLoginWithNextcloud) ...[
-                    Text(AppLocalizations.of(context).loginWorksWith),
                     const SizedBox(
                       height: 20,
                     ),
-                  ],
-                  const NextcloudLogo(),
-                  const SizedBox(
-                    height: 40,
-                  ),
-                  if (platform.canUseCamera) ...[
-                    ExcludeSemantics(
-                      child: Center(
-                        child: Text(AppLocalizations.of(context).loginUsingQrcode),
-                      ),
-                    ),
-                    IconButton(
-                      tooltip: AppLocalizations.of(context).loginUsingQrcode,
-                      icon: const Icon(
-                        Icons.qr_code_scanner,
-                        size: 50,
-                      ),
-                      onPressed: () => const LoginQrcodeRoute().go(context),
-                    ),
+                    Text(AppLocalizations.of(context).loginWorksWith),
                     const SizedBox(
                       height: 20,
                     ),
-                    ExcludeSemantics(
-                      child: Center(
-                        child: Text(AppLocalizations.of(context).loginUsingServerAddress),
-                      ),
-                    ),
+                    const NextcloudLogo(),
                   ],
+                  const SizedBox(
+                    height: 60,
+                  ),
                   Form(
                     key: _formKey,
                     child: TextFormField(
                       focusNode: _focusNode,
-                      decoration: const InputDecoration(
+                      decoration: InputDecoration(
```

Comment:
We should also provide a submit button as not everyone knows the submit functionality of the keyboard (a11y)